Workbooks/WorkbooksMetadata.json

```diff
@@ -1142,15 +1142,15 @@
     "provider": "IntSights Cyber Intelligence"
   },
   {
-    "workbookKey": "DarktraceSummaryWorkbook",
+    "workbookKey": "AIADarktraceSummaryWorkbook",
     "logoFileName": "Darktrace.svg",
     "description": "A workbook containing relevant KQL queries to help you visualise the data in model breaches from the Darktrace Connector",
     "dataTypesDependencies": [ "CommonSecurityLog" ],
     "dataConnectorsDependencies": [],
-    "previewImagesFileNames": [ "DarktraceSummaryWhite.png", "DarktraceSummaryBlack.png" ],
+    "previewImagesFileNames": [ "AIA-DarktraceSummaryWhite.png", "AIA-DarktraceSummaryBlack.png" ],
```

Comment:
These images (AIA-DarktraceSummaryWhite.png", "AIA-DarktraceSummaryBlack.png) are not available  in the preview folder that' why test cases are failing so could you please rectify it.

Thanks,
Aditi Maheshwari

Reply:
changed